packages/kolibri-common/components/SidePanelModal/index.vue

```diff
@@ -130,6 +130,11 @@
         required: false,
         default: false,
       },
+      addBottomBorder: {
```

Comment:
@marcellamaki another discrepancy in the side panels :laughing:. In the 0.18 designs we didn't have a separator line below the header in any of the frames in figma. But in the implementation of this component, we do have this line. I think for consistency we should have or should not have this line for all the side panels of at least the same section. So either option, or we show this line for the SectionSidePanel component, or we also remove this line from the resource selection side panel (or we remove this line within this component for all side panels), what do you think?